inst/demo-pipe.Rmd

```diff
@@ -424,3 +424,39 @@ sl %>% st_new() %>% st_sizes(row = 2  ) %>% stable() %>% st_asis()
 ```{r}
 data %>% st_new() %>% st_sizes(col = 20) %>% stable() %>% st_asis()
 ```
+
+\clearpage
+
+# Render to long table
+
+```{r}
+longd <- bind_rows(data, data, data, data)
+
+longd %>% 
+  st_new() %>% 
+  st_files(r = "demo-pipe.Rmd") %>%
+  st_notes("This example demonstrates how to make long table") %>%
+  st_args(lt_cap_text = "Adding a caption to the table") %>% 
+  stable_long() %>% 
+  st_asis()
+```
+
+\clearpage 
+
+# Pass any argument to stable
+
+Rather than passing dots to `stable()` or `stable_long()`, you can 
+use `st_args()`.  But note that this is destructive to previous work; it assumes 
+you are completely specifying the contents for a given argument.  See this 
+example,
```

Comment:
I appreciate the functionality but is this something we  expect people to need often? I'm struggling to think of when this might be needed... would it be worth suggesting that we recommending using one or the other, as you did above with 
>It is recommended to use either [st_units()] or [st_args()] but not both.

Reply:
@KatherineKayMRG I think if people want to keep with the pipe approach, they can just keep issuing calling `st_args()` with new arguments.  We can't prevent people from passing stuff directly to `stable()` and I wanted to be transparent about what `st_args()` was doing.

Reply:
Oh I see, that makes sense

Reply:
I pushed some text saying you shouldn't need to do both.

Reply:
thanks :)